libnavigation-core/src/main/java/com/mapbox/navigation/core/formatter/MapboxDistanceUtil.kt

```diff
@@ -193,7 +198,11 @@ object MapboxDistanceUtil {
         locale: Locale
     ): FormattedDistanceData {
         val resources = context.applicationContext.resourcesWithLocale(locale)
-        val unitStringSuffix = getUnitString(resources, getLargeTurfUnitType(unitType))
+        val unitStringSuffix = getUnitString(resources, getLargeTurfUnitType(unitType)).also {
+            if (it.isEmpty()) {
+                logW("Failed to get resource for unit: $unitType. and locale: $locale")
```

Comment:
+1 can see that the crash is potentially coming from specific language and this log would tell us which language.

But if it's a try catch, how will we ever see this log?

Reply:
We have meetings scheduled with the customer so I plan on asking them to look for these logging statements and communicate them back to us.

Reply:
I believe we could make a parameterized unit test for many languages, and then see which one causes it

Reply:
It's an information gathering kind of fix in conjunction with the customer so the root issue can be discovered and addressed.